crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -59,15 +59,18 @@ pub const PREPASS_BINDINGS_SHADER_HANDLE: HandleUntyped =
 pub const PREPASS_UTILS_SHADER_HANDLE: HandleUntyped =
     HandleUntyped::weak_from_u64(Shader::TYPE_UUID, 4603948296044544);
 
-pub struct PrepassPlugin<M: Material>(PhantomData<M>);
+/// Sets up everything required to use the prepass pipeline.
+///
+/// This does not add the actual prepasses, see [`PrepassPipeline`] for that.
```

Comment:
Is this supposed to refer to PrepassPlugin?
```suggestion
/// This does not add the actual prepasses, see [`PrepassPlugin`] for that.
```

Reply:
After this is clarified, LGTM and we can merge.